[conf_handling] New ability - modify configuration line for freshclam & clamd (#52)
cisagov/ansible-role-clamav: README.md

| Variable | Description | Default | Required |
|----------|-------------|---------|----------|

> | clamav_clamd_conf | A dictionary of values to set in the clamd configuration file. | `{}` | No |

Reviewer: Since we have `clamav_configuration_backup` I would rather use the long form for consistency.

```suggestion
| clamav_clamd_configuration | A dictionary of values to set in the clamd configuration file. | `{}` | No |
```

| clamav_configuration_backup | Whether or not to backup configuration files before changing. | `false` | No |
| clamav_cron_frequency | The frequency of ClamAV scanning.  Must be one of: `hourly`, `daily`, `weekly`, or `monthly`. | `weekly` | No |

> | clamav_freshclam_conf | A dictionary of values to set in the freshclam configuration file. | `{}` | No |

Reviewer: Since we have `clamav_configuration_backup` I would rather use the long form for consistency.

```suggestion
| clamav_freshclam_configuration | A dictionary of values to set in the freshclam configuration file. | `{}` | No |
```

### Example ###

```yaml

> clamav_freshclam_conf:

Reviewer: With my previous comment applied.

```suggestion
clamav_freshclam_configuration:
```

| Variable | Description | Default | Required |
|----------|-------------|---------|----------|
| clamav_cron_frequency | The frequency of ClamAV scanning.  Must be one of: `hourly`, `daily`, `weekly`, or `monthly`. | `weekly` | No |
| clamav_freshclam_conf | Dictionnary of values to set in freshclam configuration file  | false | No |
| clamav_clamd_conf | Dictionnary of values to set in clamd configuration file  | false | No |

> | clamav_configuration_backup | Shall we backup configuration files when changed | false | No |

Reviewer 2: Clean up the language and grammar.

```suggestion
| clamav_freshclam_conf | A dictionary of values to set in the freshclam configuration file. | false | No |
| clamav_clamd_conf | A dictionary of values to set in the clamd configuration file. | false | No |
| clamav_configuration_backup | Whether or not to backup configuration files before changing. | false | No |
```

> | clamav_configuration_backup | Whether or not to backup configuration files before changing. | false | No |

Reviewer: These should be sorted. I also changed the default of the two values that are supposed to be dictionaries and made sure all values are code formatted.

```suggestion
| clamav_clamd_conf | A dictionary of values to set in the clamd configuration file. | `{}` | No |
| clamav_configuration_backup | Whether or not to backup configuration files before changing. | `false` | No |
| clamav_cron_frequency | The frequency of ClamAV scanning.  Must be one of: `hourly`, `daily`, `weekly`, or `monthly`. | `weekly` | No |
| clamav_freshclam_conf | A dictionary of values to set in the freshclam configuration file. | `{}` | No |
```

Reviewer 2: In @mcdonnnj's comment above, "sorted" means "sorted alphabetically".

### Example ###

```yaml
clamav_freshclam_conf:
  DatabaseMirror: ['db.local.clamav.net', 'database.clamav.net']
  Bytecode: 'true'
  PrivateMirror: None
```

> would change :

Reviewer 2: Remove unnecessary space.

```suggestion
would change:
```

> ```shell

Reviewer 2: I think this will look better with `properties` syntax highlighting:
```suggestion
```properties
```

  ...
  DatabaseMirror foo.bar.com
  DatabaseMirror bar.baz.com
  PrivateMirror private.mirror.local
  Bytecode false
  ...
```

> to :

Reviewer 2: Remove unnecessary space:
```suggestion
to:
```

> ```shell

Reviewer 2: I think this will look better with `properties` syntax highlighting:
```suggestion
```properties
```
